### PR TITLE
added the protectRoute middleware for non authenticated user

### PR DESCRIPTION
**Pull request by Akash-gahlot**
Issue : #118 

@shreya024 

 IMPORTANT --> For testing purpose i have  set isLogged value to false in protectRoute and did a small modification in aboutus route so a user without login cannot access the aboutus page , because we dont have any route apart from these /home,/aboutus,/contactus,/login,/signup so i choose aboutus page for testing purpose.
we can revert the aboutus code changes later after testing the protectRoute working.

https://github.com/SrijanShovit/carbonops-v2/assets/71133465/4ddc0140-ab73-45d8-8e54-f6f3d25d81be

### CODE REVIEW

**Review by SrijanShovit — COMMENTED** [review]
Hello Akash-gahlot, thanks for rising a Pull request, your contribution is valuable to us. The maintainers will review this Pull Request and provide feedback as soon as possible. Keep the great work up!<p></p>

File: app/aboutus/page.js [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+      <Team />
     </div>
-  )
-}
-
-export default About
\ No newline at end of file
+  );
+};
+//for testing purpose added the protectRoute please remove the protectRoute from here to access this page.
```
Comment (shreya024) [inline app/aboutus/page.js:19]: Could you test it on /calculate,rather than on the /aboutus

**Review by shreya024 — CHANGES_REQUESTED** [review]

File: app/aboutus/page.js (hunk 1 above)
Comment (Akash-gahlot) [reply app/aboutus/page.js]: completed the required fixed @shreya024 , could you please validate the changes and approve.

**Review by shreya024 — CHANGES_REQUESTED** [review]
@Akash-gahlot Resolve the vercel error. Everything else LGTM!

**Comment by Akash-gahlot** [comment]
@shreya024 when i clicked on vercel error  so it shows commit  is pending for approval from @SrijanShovit  ,
can you confirm am i correct here or something from my side only is pending?
please have look on this below image.

image

**Comment by shreya024** [comment]
@Akash-gahlot Resolve the conflicts and push again

**Pull request closed without merging**